addon/hint/show-hint.js

```diff
@@ -229,14 +229,24 @@
       elt.hintId = i;
     }
 
+    var container = completion.options.container || ownerDocument.body;
     var pos = cm.cursorCoords(completion.options.alignWithWord ? data.from : null);
     var left = pos.left, top = pos.bottom, below = true;
+
+    if(container !== ownerDocument.body) {
+      var offsetParentPosition = container.offsetParent.getBoundingClientRect();
```

Comment:
Hi, I tried this out locally and feel like this line should be `container.getBoundingClientRect();` to get the hint to show up at the exact same position as before. I maybe wrong, but it didn't work for me as is.

Reply:
I guess that depends on whether `container` is positioned or not—I guess we'd want the `offsetParent` of a _child_ of the container.